examples/dump_nl80211_station.rs

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+
+use futures::stream::TryStreamExt;
+
+fn main() {
+    let rt = tokio::runtime::Builder::new_current_thread()
+        .enable_io()
+        .build()
+        .unwrap();
+    rt.block_on(dump_station(3));
```

Comment:
Can you take the interface index from `arg[1]`?

You may take this as example: https://github.com/rust-netlink/genetlink/blob/main/examples/dump_family_policy.rs